source/includes/extracts-agg-stages.yaml

```diff
@@ -176,8 +176,25 @@ content: |
           .. note:: 
 
              ``$search`` is only available for MongoDB Atlas clusters, 
-             and is not available for self-managed deployments.
+             and is not available for self-managed deployments. To 
+             learn more, see :atlas:`Atlas Search Aggregation Pipeline 
+             Stages </reference/atlas-search/query-syntax>`.
 
+      * - :pipeline:`$searchMeta`
+
+        - Returns different types of metadata result documents for the 
+          :atlas:`Atlas Search </atlas-search>` query against an 
+          :atlas:`Atlas </reference/atlas-search/query-syntax/>` 
+          collection. 
+
+          .. note:: 
+
+             ``$searchMeta`` is only available for MongoDB Atlas 
+             clusters running MongoDB v4.4.9 or higher, and is not 
+             available for self-managed deployments. To learn more, see 
```

Comment:
Might also be good to mention that the feature flag needs to be turned on right now.

Reply:
very important.

Reply:
It isn't critical at this stage though because all the customers are being handheld through the process and the page will be unlisted.